hermes-api/src/main/java/pl/allegro/tech/hermes/api/AuthenticationData.java

```diff
@@ -0,0 +1,9 @@
+package pl.allegro.tech.hermes.api;
+
+public interface AuthenticationData {
+
+    public String getUsername();
```

Comment:
no need for `public` in interface, also shouldn't `anoymize()` be a part of this interface?